src/Azure/Orleans.Persistence.AzureStorage/Providers/Storage/AzureTableStorageOptions.cs

```diff
@@ -0,0 +1,92 @@
+using System.Collections.Generic;
+using Microsoft.Extensions.Options;
+using Newtonsoft.Json;
+using Orleans.Runtime;
+using Orleans.Runtime.Configuration;
+
+namespace Orleans.Hosting
+{
+    /// <summary>
+    /// Configuration for AzureTableGrainStorage
+    /// </summary>
+    public class AzureTableStorageOptions
+    {
+        /// <summary>
+        /// Gets or sets a unique identifier for this service, which should survive deployment and redeployment.
+        /// </summary>
+        public string ServiceId { get; set; } = string.Empty;
+
+        /// <summary>
+        /// Azure data connection string
+        /// </summary>
+        public string DataConnectionString { get; set; }
+
+        /// <summary>
+        /// Table name where grain stage is stored
+        /// </summary>
+        public string TableName { get; set; } = DEFAULT_TABLE_NAME;
+        public const string DEFAULT_TABLE_NAME = "OrleansGrainState";
+
+        /// <summary>
+        /// Indicates if grain data should be deleted or reset to defaults when a grain clears it's state.
+        /// </summary>
+        public bool DeleteStateOnClear { get; set; } = false;
```

Comment:
I think we should document the default value. something like "off by default"

Reply:
As a general pattern, we don't tend to document defaults if they are the same as the default for the type.  The default for a bool is false, so assumed default for the type, given no stated default, is correct.

Not opposed to this suggestion, but we should be consistent.